rosdep/python.yaml

```
@@ -29,6 +29,10 @@ adafruit-pca9685-pip:
   ubuntu:
     pip:
       packages: [adafruit-pca9685]
+aioraven:
+  '*':
+    pip:
+      packages: [aioraven]
```

Keys which contain only pip rules should have the '-pip' suffix.

```suggestion
aioraven-pip:
  '*':
    pip:
      packages: [aioraven]
```